subjects/java/checkpoints/steady-sequence/README.md

```diff
@@ -0,0 +1,53 @@
+## Steady Sequence
+
+### Instructions
+
+Create a class `SteadySequence` that provides a method to find the length of the longest consecutive elements sequence in an unsorted array of integers.
+
```

Comment:
What is meant by "longest consecutive elements sequence"? It would be nice to have a definition to avoid having to understand what is expected from the examples

Reply:
done